README.md

```diff
@@ -30,7 +32,9 @@ are not published but can be build locally if required.
 The release process is automated by GitHub actions.
 
 # Known Compatible Docker Versions
-* 20.10.23
+
+- 20.10.23
 
 # Contributing
+
 If you would like to make a Contribution, we have all the details for doing that [here](CONTRIBUTING.md)
```

Comment:
This `CONTRIBUTING.md` file is pretty outdated. We can delete this and link to `https://gchq.github.io/gaffer-doc/latest/dev/ways-of-working/` instead for details on how to contribute.